x-pack/plugins/uptime/server/lib/adapters/framework/adapter_types.ts

```diff
@@ -44,13 +44,12 @@ export type UMSavedObjectsQueryFn<T = any, P = undefined> = (
 ) => Promise<T> | T;
 
 export interface UptimeCoreSetup {
-  route: IRouter;
+  router: IRouter;
 }
 
 export interface UptimeCorePlugins {
-  savedObjects: SavedObjectsLegacyService<any>;
+  features: PluginSetupContract;
   usageCollection: UsageCollectionSetup;
-  xpack: any;
 }
```

Comment:
NIT: how are these 'adapter' types ? Should probably be moved to the root plugin's file or close.

Reply:
Getting rid of those things and referencing the plugins directly in the future 👍